prow/cmd/mkbuild-cluster/main.go

```diff
@@ -103,6 +107,21 @@ func main() {
 	}
 }
 
+// createDefaultClient creates and returns a default kubernetes clientset
+func createDefaultClient() (*kubernetes.Clientset, error) {
+	config, err := clientcmd.NewNonInteractiveDeferredLoadingClientConfig(clientcmd.NewDefaultClientConfigLoadingRules(), &clientcmd.ConfigOverrides{}).ClientConfig()
```

Comment:
Can we have this get it for a specific context?

Reply:
`ConfigOverrides` accepts a `CurrentContext` field which can be used to set the context.

Reply:
@fejta - I updated. Now I am setting the context to what is set by `get-credentials` falling back to `current-context` to correctly honor with `--change-context` flag.